src/Shared/UnitTests/TypeLoader_Dependencies_Tests.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System;
+using System.IO;
+using Microsoft.Build.Shared;
+using Microsoft.Build.SharedUtilities;
+using Xunit;
+
+namespace Microsoft.Build.UnitTests
+{
+    public class TypeLoader_Dependencies_Tests
```

Comment:
@cdmihai I could, but these tests cover a somewhat different scenario than TypeLoader_Tests--loading dependencies, rather than "top level" task assemblies. I generally prefer a larger number of smaller tests, rather than fewer tests that exercise multiple concepts.